src/main/java/Deadlines.java

```diff
@@ -0,0 +1,27 @@
+public class Deadlines extends Task {
+    protected String taskType = "[D]";
+    protected String deadline;
+
+    public Deadlines(String description, String d) {
```

Comment:
```suggestion
    public Deadlines(String description, String deadline) {
```

Should this [not be a single variable name](https://se-education.org/guides/conventions/java/intermediate.html#naming)?